ocs_ci/framework/__init__.py

```diff
@@ -28,6 +28,7 @@ class Config:
     UPGRADE: dict = field(default_factory=dict)
     FLEXY: dict = field(default_factory=dict)
     UI_SELENIUM: dict = field(default_factory=dict)
+    COMPONENTS: dict = field(default_factory=dict)
```

Comment:
We should document this new config section [here](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md)

Reply:
ack, I will do that

Reply:
still missing documentation

Reply:
done